storage/s3-sdk/list_gcs_buckets.py

```diff
@@ -21,11 +21,19 @@
 
 
 def list_gcs_buckets(google_access_key_id, google_access_key_secret):
```

Comment:
Don't forget docstrings for the parameters.

Reply:
See PR #9947